internal/controllers/bundledeployment/bundledeployment.go

```diff
@@ -310,12 +332,30 @@ func (c *controller) reconcile(ctx context.Context, bd *rukpakv1alpha2.BundleDep
 		},
 	}
 
-	rel, state, err := c.getReleaseState(cl, bd, chrt, values, post)
+	// TODO: update this to return desired release as well
```

Comment:
Is this todo going to be resolved before merging this?

Reply:
Ah, I think that should have already been addressed... I'll double check